modules/friendlytag.js

```diff
@@ -186,14 +186,21 @@ Twinkle.tag.callback = function friendlytagCallback() {
 		case 'redirect':
 			Window.setTitle('Redirect tagging');
 
-			form.append({ type: 'header', label: 'Spelling, misspelling, tense and capitalization templates' });
-			form.append({ type: 'checkbox', name: 'tags', list: Twinkle.tag.spellingList });
-
-			form.append({ type: 'header', label: 'Alternative name templates' });
-			form.append({ type: 'checkbox', name: 'tags', list: Twinkle.tag.alternativeList });
-
-			form.append({ type: 'header', label: 'Miscellaneous and administrative redirect templates' });
-			form.append({ type: 'checkbox', name: 'tags', list: Twinkle.tag.administrativeList });
+			var i = 1;
+			$.each(Twinkle.tag.redirectList, function(groupName, group) {
+				form.append({ type: 'header', id: 'tagHeader' + i, label: groupName });
+				var subdiv = form.append({ type: 'div', id: 'tagSubdiv' + i++ });
+				$.each(group, function(subgroupName, subgroup) {
+					subdiv.append({ type: 'div', label: [ Morebits.htmlNode('b', subgroupName) ] });
+					subdiv.append({
+						type: 'checkbox',
+						name: 'redirectTags',
+						list: subgroup.map(function (item) {
+							return { value: item.tag, label: '{{' + item.tag + '}}: ' + item.description };
```

Comment:
Bug: Lost your extra input data stored in subgroup
Suggest: `return { value: item.tag, label: '{{' + item.tag + '}}: ' + item.description, subgroup: item.subgroup };`

Reply:
Thanks for keeping me honest @Xi-Plus, done (plus another from a different PR ) in #1055